gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/tasks/CheckPegasusSnapshotTask.java

```diff
@@ -76,6 +84,15 @@ public void checkPegasusSnapshot()
       javaExecSpec.args("--report", reportOutput);
       javaExecSpec.args(_previousSnapshotDirectory);
       javaExecSpec.args(_currentSnapshotDirectory);
+      if (_isExtensionSchema)
+      {
+        javaExecSpec.args("--extensionSchema");
+      }
+      else if(hasSchemaAnnotationHandler())
+      {
+        javaExecSpec.args("--jarPath", _handlerJarPath);
```

Comment:
I will recommend to use longer name here, `--handlerJarPath` and `handlerClassName`. We don't need to be concise for this particular use case, but need to be explicit (for example later do a code search would be easier)

Reply:
fixed